### PR TITLE
add FR holder for equivalent to PR19774 that was recently merged, tra…

### PR DESCRIPTION
**Pull request by spurin**
Hi All,

I recently made the equivalent change on the English equivalent to this page, it was merged under pull request - https://github.com/kubernetes/website/pull/19774

In the spirit of consistency, I can see that there is an equivalent locale version for French.  Unfortunately, French is not a language I know and I don't think an online translation service will do this the justice it deserves, so for now, I have put the equivalent holder in place.

Could someone please provide an appropriate translation in the comments and I will update accordingly for further review.

For convenience, the text that needs translation is -

"Helper programs relating to the volume type may be required for consumption of a PersistentVolume within a cluster.  In this example, the PersistentVolume is of type NFS and the helper program /sbin/mount.nfs is required to support the mounting of NFS filesystems."

Many Thanks

James

### CODE REVIEW

**Comment by spurin** [comment]
@anthonydahanne @abuisine - If either of you can offer a translation of the sentence above it would be appreciated.  Thanks

**Comment by anthonydahanne** [comment]
Hello @spurin 

```
Des logiciel additionnels supportant un type de montage de volume pourraient être nécessaires afin d'utiliser un PersistentVolume depuis un cluster. Dans l'exemple d'un PersistentVolume de type NFS, le logiciel additionnel `/sbin/mount.nfs` est requis pour permettre de monter des systèmes de fichiers de type NFS.
```

that should do it ^

Thanks for letting us know!

**Comment by spurin** [comment]
Thanks @anthonydahanne for the translation, I initially tried to thank you in the commit message, not realising that this wasn't allowed.  I've since reverted back to head and have put your translation as a new commit.

**Comment by spurin** [comment]
For convenience, link to the changes in the deploy-preview - https://deploy-preview-19807--kubernetes-io-master-staging.netlify.com/fr/docs/concepts/storage/persistent-volumes/#volumes-persistants

**Comment by sftim** [comment]
This change looks good to me.

**Comment by spurin** [comment]
Hi @abuisine , could you review, thanks.

**Comment by spurin** [comment]
Adding @kubernetes/sig-docs-fr-reviews for review

**Comment by spurin** [comment]
adding @sig-docs-fr-reviews

**Comment by spurin** [comment]
@anthonydahanne do you know how I could request an alternative FR reviewer to close this one out , I think @abuisine may be busy (remarks implied as constructive only).

Thanks

**Comment by sftim** [comment]
@kubernetes/sig-docs-fr-reviews could you take a look?

File: content/fr/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -334,6 +334,10 @@ spec:
     server: 172.17.0.2
 ```
 
+{{< note >}}
+Des logiciel additionnels supportant un type de montage de volume pourraient être nécessaires afin d'utiliser un PersistentVolume depuis un cluster. Dans l'exemple d'un PersistentVolume de type NFS, le logiciel additionnel `/sbin/mount.nfs` est requis pour permettre de monter des systèmes de fichiers de type NFS.
```
Comment (remyleone) [inline content/fr/docs/concepts/storage/persistent-volumes.md:338]:
```suggestion
Des logiciels additionnels supportant un type de montage de volume pourraient être nécessaires afin d'utiliser un PersistentVolume depuis un cluster.
Dans l'exemple d'un PersistentVolume de type NFS, le logiciel additionnel `/sbin/mount.nfs` est requis pour permettre de monter des systèmes de fichiers de type NFS.
```

**Comment by remyleone** [comment]
done

**Comment by spurin** [comment]
> done

Thanks @remyleone

**Comment by spurin** [comment]
@kubernetes/sig-docs-fr-owners would you mind assigning an alternative assignee please.  The reviews and feedback from both @anthonydahanne and @remyleone has been implemented

Thanks

**Pull request merged**